DetectorDescription/DDCMS/plugins/MuonNumberingESProducer.cc

```diff
@@ -0,0 +1,83 @@
+// -*- C++ -*-
+//
+// Package:    DetectorDescription/MuonNumberingESProducer
+// Class:      MuonNumberingESProducer
+// 
+/**\class MuonNumberingESProducer
+
+ Description: [one line class summary]
+
+ Implementation:
+     [Notes on implementation]
```

Comment:
Shouldn't comment lines 8 and 11 be filled in?

Reply:
it will be resolved in the next PR